packages/solana-client/src/GTransaction.ts

```diff
@@ -149,32 +152,41 @@ export namespace GTransaction {
         accounts: accounts.map((a) => a.address),
       })),
     };
+
+    return GTransaction.sign({
+      signers,
+      gtransaction,
+    });
   };
 
   export const sign = ({
-    secretKey,
+    signers,
     gtransaction,
   }: {
     gtransaction: GTransaction;
-    secretKey: Buffer | Uint8Array;
+    signers: Array<GKeypair>;
```

Comment:
It feels a bit strict to require `GKeypair` when we just need `{ secretKey: Uint8Array }`. Maybe we can do that and `GKeypair` will also be accepted?